tests/unit/hubble/test_helper.py

```diff
@@ -91,7 +91,7 @@ def test_unpack_package_unsupported(tmpdir):
 
 def test_install_requirements():
     os.environ['DOWNLOAD']="download"
-    os.environ['TEST_TOKEN']="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"
+    os.environ['TEST_TOKEN']="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"
```

Comment:
Is this real token?

Reply:
yes it is,  i will remove it

Reply:
You can use other parts to test the functionality. For example, have a 

- `DOMAIN=github.com` 
- `docsqalib @ git+https://${DOMAIN}/somepublic/repo.git`

to simulate the case and avoid the real token.